go/tools/bzltestutil/wrap.go

```diff
@@ -61,6 +62,7 @@ func shouldAddTestV() bool {
 func Wrap(pkg string) error {
 	var jsonBuffer bytes.Buffer
 	jsonConverter := NewConverter(&jsonBuffer, pkg, Timestamp)
+	pipeRead, pipeWrite := io.Pipe()
```

Comment:
Why is a pipe needed/chosen here over the following?
```
cmd.Stderr = io.MultiWriter(os.Stderr, jsonConverter)
cmd.Stdout = io.MultiWriter(os.Stdout, jsonConverter)
```

Reply:
This is what I tried first, but it's racy since we have two streams writing to the same underlying `bytes.Buffer`. The `io.Pipe` is thread-safe.